.github/workflows/test.yml

```
@@ -0,0 +1,316 @@
+
+env:
+  FORCE_COLOR: 1
+  NODE_NO_WARNINGS: 1
+
+jobs:
+  setup:
+    name: 🏗️ Setup
+    runs-on: ubuntu-latest
+    outputs:
+      node-version: ${{ steps.setup.outputs.node-version }}
+      pnpm-version: ${{ steps.setup.outputs.pnpm-version }}
```

actionlint (1.7.7)

property "pnpm-version" is not defined in object type {cache-hit: string; node-version: string}

(expression)